drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -58,9 +61,15 @@ static void dsa_ksz8xxx_write_reg(const struct ksz8xxx_data *pdev,
 		.count = 1
 	};
 
+#if CONFIG_DSA_KSZ8463
+	buf[0] = KSZ8XXX_SPI_CMD_WR | (reg_addr & 0x7FF) >> 4;
```

Comment:
Consider some macro definitions to avoid magic numbers

Reply:
fixed